src/metatensor/models/utils/distributed/slurm.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import hostlist
+
+
+def is_slurm():
+    return "SLURM_JOB_ID" in os.environ
+
+
+def is_slurm_main_process():
+    return os.environ["SLURM_PROCID"] == "0"
+
+
+class DistributedEnvironment:
```

Comment:
This is stolen from LUMI and I have no clue what it does... Pehaps I can provide a link to LUMI (but the class is also undocumented there)
https://github.com/Lumi-supercomputer/lumi-reframe-tests/blob/main/checks/apps/deeplearning/pytorch/src/pt_distr_env.py

Reply:
=/ I don't like this, because this mean we have no way of fixing the code when it breaks. Is there some documentation on what environment variables need to be set for `torch.distributed`?